.gitignore

```diff
@@ -3,7 +3,6 @@ VirtualEnv
 */.DS_Store
 *.DS_Store
 .DS_Store
-media/
```

Comment:
People should not commit the content of the media folder so you should add:
`/project/media/*`

But that will ignore `.gitkeep` file.

So you should also set the `.gitignore` file such that it ignores the content of `media` folder except for `.gitkeep` file. You can look at what is done for `staticfiles` folder in `.gitignore` for this purpose.